tests/test_app/cms_plugins.py

```diff
@@ -0,0 +1,13 @@
+from cms.plugin_base import CMSPluginBase
+from cms.plugin_pool import plugin_pool
+from django.utils.translation import gettext_lazy as _
+
+from .models import Container
+
+
+@plugin_pool.register_plugin
+class ContainerPlugin(CMSPluginBase):
+    name = _("Container")
+    model = Container
```

Comment:
```suggestion
```
This is not needed. If you do not have custom fields in a plugin you need not provide a model